rte/mo_rte_lw.F90

```diff
@@ -70,16 +71,21 @@ function rte_lw(optical_props, top_at_1, &
               target, optional, intent(in   ) :: inc_flux    ! incident flux at domain top [W/m2] (ncol, ngpts)
     integer,          optional, intent(in   ) :: n_gauss_angles ! Number of angles used in Gaussian quadrature
                                                                 ! (no-scattering solution)
+    real(wp), dimension(:,:),   &
+                      optional, intent(in   ) :: lw_Ds
     character(len=128)                        :: error_msg   ! If empty, calculation was successful
     ! --------------------------------
     !
     ! Local variables
     !
     integer :: ncol, nlay, ngpt, nband
     integer :: n_quad_angs
+    integer :: iquad
     integer :: icol, iband, igpt
     real(wp), dimension(:,:,:), allocatable :: gpt_flux_up, gpt_flux_dn
     real(wp), dimension(:,:),   allocatable :: sfc_emis_gpt
+    real(wp) :: lw_Ds_wt
```

Comment:
We should check that `n_quad_angle == 1` or isn't supplied if `lw_Ds` is supplied - they are mutually exclusive.

Reply:
n_quad_angs defaults to one but is reassigned to the value of n_gauss_angles if that variable is present.  Where does it make sense to put this check?

Reply:
Didn't resolve - `n_gauss_angles` is ignored if `lw_Ds` are supplied